index.html

```diff
@@ -1,270 +1,222 @@
+            <img src="images/arrows/Separator Bottom.png" alt="horizontal arrow separater" />
           </li>
-
+        </ul>
+        <ul id="skills">
           <li>
-            <a href="#"><img src="images/icons/Twitter.png" alt="twitter-icon" class="twitter"></a>
+            <p class="skill_text">skills</p>
           </li>
-
+          <img src="images/arrows/Icon - Chevron - Right - Larger.png" alt="right_arrow_icon" class="right_arrow" />
           <li>
-            <a href="#"><img src="images/icons/Medium.png" alt="medium-icon"></a>
+            <img src="images/arrows/Separator Bottom.png" alt="horizontal arrow separater" />
           </li>
         </ul>
-        <button type="button" class="get_resume">Get my resume</button>
       </div>
-
-      <div class="skill_set">
-        <p class="language-text">Language</p>
-        <img
-          src="images/arrows/down arrow.png"
-          alt="down arrow"
-          class="down_arrow"
-        />
-
-        <div class="language_list_container">
-          <ul class="language_list">
-            <li class="javascrip_1">
-              <img
-                src="images/lg-banners/Ellipse 1.png"
-                alt="Javascript-logo"
-                class="js_arrow"
-              />
-              <p class="javascript_text">Javascript</p>
-            </li>
-            <li class="html_1">
-              <img
-                src="images/lg-banners/Ellipse 1 (2).png"
-                alt="HTML-logo"
-                class="html_arrow"
-              />
-              <p class="html_text">HTML</p>
-            </li>
-            <li class="css_1">
-              <img
-                src="images/lg-banners/Ellipse 1 (3).png"
-                alt="CSS-logo"
-                class="css_arrow"
-              />
-              <p class="css_text">CSS</p>
-            </li>
-          </ul>
-
-          <ul id="framworks">
-            <li>
-              <p class="frameworks_text">Frameworks</p>
-            </li>
-            <img
-              src="images/arrows/Icon - Chevron - Right - Larger.png"
-              alt="right_arrow_icon"
-              class="right_arrow1"
-            />
-            <li>
-              <img
-                src="images/arrows/Separator Bottom.png"
-                alt="horizontal arrow separater"
-              />
-            </li>
-          </ul>
-          <ul id="skills">
-            <li>
-              <p class="skill_text">skills</p>
-            </li>
-            <img
-              src="images/arrows/Icon - Chevron - Right - Larger.png"
-              alt="right_arrow_icon"
-              class="right_arrow"
-            />
-            <li>
-              <img
-                src="images/arrows/Separator Bottom.png"
-                alt="horizontal arrow separater"
-              />
-            </li>
-          </ul>
-        </div>
-      </div> 
+    </div>
 
       <div class="contact_container">
```

Comment:
- [ ] Please add required to the text-area as required for validation purpose

![image](https://user-images.githubusercontent.com/75126481/230060398-447a6c0f-988d-450a-83c1-ef647fefb303.png)